charts/pega/README.md

```diff
@@ -805,7 +805,7 @@ Upgrade type    | Description
 ---             | ---
 `zero-downtime` |  If applying any patch or upgrading from 8.4.2 and later, use this option to minimize downtime. This patch or upgrade type migrates the rules to your designated "new rules schema", uses the temporary data schema to host some required data-specific tables, and patches or upgrades the rules to the new version with zero-downtime. With the new rules in place, the process performs a rolling reboot of your nodes, patches or upgrades any required data schema, and redeploys the application using the new rules.
 `custom` |  Use this option for any upgrade in which you complete portions of the upgrade process in steps. Supported upgrade steps are: `enable_cluster_upgrade` `rules_migration` `rules_upgrade` `data_upgrade` `disable_cluster_upgrade`. To specify which steps to include in your custom upgrade, specify them in your pega.yaml file using the `upgrade.upgradeSteps` parameter.
-`out-of-place-rules` | Use this option to run an out-of-place upgrade of rules that runs against a migrated copy of the rules schema. This schema will become the rules schema after your upgrade is complete.
+`out-of-place-rules` | Use this option to migrate a copy of the rules to a new schema and run an out-of-place upgrade in that copied schema. This schema will become the rules schema after your upgrade is complete.
```

Comment:
can you please clarify that this is the new rules schema, not just "a" schema?
Use this option to migrate a copy of the rules to a new rules schema and run an out-of-place upgrade of the rules in that copied schema. This schema will become the rules schema after your upgrade is complete.

Reply:
fixed